web/templates/settings/custom_user_profile_field.hbs

```diff
@@ -1,5 +1,6 @@
 <div class="custom_user_field" name="{{ field.name }}" data-field-id="{{ field.id }}">
-    <label class="inline-block" for="{{ field.name }}" class="title">{{ field.name }}</label>
+    <label class="inline-block {{#if field.required}}required_field{{/if}}" for="{{ field.name }}" class="title">{{ field.name }} </label>
+    {{#if field.required}}<span class="required-symbol">*</span>{{/if}}
```

Comment:
Is the right place to put the space inside the `label` element or inside this `field.required` clause? I'm not sure why you're adding one just before the `</label>` specifically.